src/test/java/org/fcrepo/importexport/integration/AbstractResourceIT.java

```diff
@@ -50,7 +50,7 @@
  * @author awoods
  * @since 2016-09-01
  */
-abstract class AbstractResourceIT {
+public abstract class AbstractResourceIT {
```

Comment:
I couldn't execute the ITs in my IDE without this change, although they ran via commandline